cmd/juju/controller/destroy.go

```diff
@@ -177,6 +189,38 @@ func (c *destroyCommand) Init(args []string) error {
 	return c.destroyCommandBase.Init(args)
 }
 
+// getModelNames gets slice of model names from modelData.
+func getModelNames(data []modelData) []string {
+	return transform.Slice(data, func(f modelData) string {
+		return f.Name
+	})
+}
+
+// printDestroyWarning prints to stdout the warning with additional info about destroying controller.
+func printDestroyWarning(ctx *cmd.Context, modelStatus environmentStatus, controllerName string, releaseStorage bool) error {
+	modelNames := getModelNames(modelStatus.models)
+	var actionStorageStr string
+	if releaseStorage {
+		actionStorageStr = "released"
+	} else {
+		actionStorageStr = "destroyed"
+	}
+	if len(modelNames) > 0 {
+		_, _ = fmt.Fprintf(ctx.Stdout, destroySysMsgWithDetails, controllerName,
```

Comment:
I think this should be split between stdout and stderr. I think the first two lines should go to Stderr (as that's the 'warning') and the remainder to stdout.